schema_based_on_diagram.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(30),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP, 
+    patient_id FOREIGN KEY, 
+    status VARCHAR(20)
+);
+
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(50),
+    name VARCHAR(50)
+);
+
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL , 
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP, 
+    medical_historY_id FOREIGN KEY 
```

Comment:
- [ ] @Team great work adding all columns for the invoices table however the `medical_historY_id` needs to be in all lower case for accurate referencing `medical_history_id`